flink-examples/doc/flink-high-error-count-alert/README.md

```diff
@@ -0,0 +1,140 @@
+# High Count Alert #
+
+The application reads apache access logs from a Pravega stream and once every 2 seconds
+counts the number of 500 responses in the last 30 seconds, and generates
+alert when the counts of 500 responses exceed 6.
+
+## Prerequistes ##
+
+1. Logstash installed, see [Install logstash](https://www.elastic.co/guide/en/logstash/5.6/installing-logstash.html).
+2. Pravega running, see [here](http://pravega.io/docs/latest/getting-started/) for instructions.
+
+Note that you can also skip logstash to demonstrate this sample. Instead, use the WordCountWriter sample to push
+apache access logs as json strings to Pravega. See details below.
+
+## Run HighCountAlerter ##
+
+```
+$ cd flink-examples/build/install/pravega-flink-examples
+$ bin/highCountAlerter [--controller tcp://127.0.0.1:9090] [--stream myscope/apacheaccess]
+```
+
+## Start Logstash with Pravega Output Plugin ##
+Copy the contents under flink-examples/doc/flink-high-error-count-alert/filters/ to the host
+where logstash is installed, e.g., in directory ~/pravega.
+update **pravega_endpoint** in ~/pravega/90-pravega-output.conf
+
+```
+output {
+    pravega {
+        pravega_endpoint => "tcp://127.0.0.1:9090"   <- update to point to your Pravega controller
+        stream_name => "apacheaccess"
+        scope => "myscope"
+    }
+}
+```
+
+Start logstash, assuming it is installed at /usr/share/logstash/bin.
+Note that sometimes it may take a minute or two for logstash to start. For troubleshooting, the logstash log files are 
+normally at /var/log/logstash. To restart, type Ctrl-C, then re-run the command.
+
+```
+$ sudo /usr/share/logstash/bin -f ~/pravega
+Sending Logstash's logs to /var/log/logstash which is now configured via log4j2.properties
+The stdin plugin is now waiting for input:
+```
+
+## Input Data via Logstash ##
+
+In the logstash window, paste apache access logs like the followings:
+```
+10.1.1.11 - peter [19/Mar/2018:02:24:01 -0400] "GET /health/ HTTP/1.1" 200 182 "http://example.com/myapp" "Mozilla/5.0"
```

Comment:
If I understand this right, this is expecting some manual input from the user. Could we also have a way to tail access logs coming through files? For example, if I start a web server locally and point logstash to the directory with the access logs, it should be able to tail it, no?

Reply:
The common practice is  logs -> filebeat -> logstash -> ...  Given the number components in the demo, I will make a docker image to set up the end to end environment.

Reply:
@fpj, I've updated the sample to read from a file. I also built a docker image to make it easier to set up the environment. I believe i had addressed all the requests from you.

While committing changes, i forgot to set signed-off-by. Had to use amend/rebase to fix them. But somehow the latest commit still complains even if there is a signed-off-by identical to others. I am puzzled and may need  some help to fix it.